modules/smart-agent_processes/README.md

```diff
@@ -103,6 +104,7 @@ the corresponding monitor configuration:
       - metricNames:
         - '*'
         - '!ps_count.processes'
+        - '!tnslsnr'
```

Comment:
please run the make again (or git reset --hard this file) to cancel changes on it

Reply:
Done